.dependabot/config.yml

```diff
@@ -1,33 +0,0 @@
-version: 1
-
-update_configs:
-
-  - package_manager: 'javascript'
-    directory: '/'
-    target_branch: main
-    update_schedule: 'live'
-    version_requirement_updates: 'increase_versions'
-
-    commit_message:
-      prefix: 'fix'
-      prefix_development: 'fix'
-      include_scope: true
-
-    allowed_updates:
-      - match:
-          update_type: 'security'
```

Comment:
Why are we getting rid of the security bumps? I though we decided on the custom CI job because we wanted to have both live security updates and weekly minor updates.

Reply:
Yeah, I know. That used to be the case, but I recently learned that GitHub now handles security independently from updates. See article in the description, that means we don't need this here. I still feel like our custom job is probably better than Dependabot for the other use case, but if this works, we can maybe try what would happen if we used it for the lockfile maintenance, too. This is somewhat of an experiment in that regard, too.